README.md

```diff
@@ -1242,6 +1243,55 @@ $ m . /example
     1 directory, 1 file
 ```
 
+### path
+
+Store a path in the [Nix][NIX] store
+and use it as an input
+for higher level builtins.
+It can be used for including paths in
+[makeScript](#makescript)
+and [makeDerivation](#makederivation)
+via `replace` and `env`
+parameters respectively.
+
+Inputs:
+
+- path (`str`):
+  path to store.
+  It must be absolute
+  within the repository.
+
+Example:
+
+```nix
+# Consider the following path within the repository: /src/nix
+
+# /path/to/my/project/makes/example/main.nix
+{ inputs
+, makeScript
+, ...
+}:
+makeScript {
+  replace = {
+    __argPath__ = path "/src/nix";
+  };
+  builder = ''
+    info Path is: __argPath__
+    info Path contents are:
+    ls __argPath__
+  '';
+  name = "example";
+}
+```
+
+```bash
+$ m . /example
+
+    [INFO] Path is: <nix-store-path>
+    [INFO] Path contents are:
+    packages.nix  sources.json  sources.nix
+```
+
```

Comment:
it would be nice to mention that using `path "/somwhere"` is faster and more reproducible than `./somewhere`

path removes spurious files and folder created while developing that introduces fragility, it also load the head once, and then reuses in all invocations